docs/displaysystem/angular.md

```diff
@@ -1,219 +0,0 @@
----
-layout: page
-title: "Angular Display System"
```

Comment:
Same as above, could you please double-check that this needs to be removed as part of R interpreter implementation?